src/com/serotonin/mango/rt/dataSource/meta/MetaPointLocatorRT.java

```diff
@@ -109,6 +110,10 @@ public void initialize(AbstractTimer timer, MetaDataSourceRT dataSource, DataPoi
         initialized = true;
 
         initializeTimerTask();
+
+        if(dataPoint.getPointValue() != null && vo.getUpdateEvent() == MetaPointLocatorVO.UPDATE_EVENT_CONTEXT_CHANGE) {
+            execute(System.currentTimeMillis(), new ArrayList<>(), true);
+        }
```

Comment:
To avoid inconsistencies, we should use the DataPointRT object passed to the initialize method:

```
        if(vo.getUpdateEvent() == MetaPointLocatorVO.UPDATE_EVENT_CONTEXT_CHANGE) {
            execute(System.currentTimeMillis(), new ArrayList<>(), true, dataPoint);
        }
```